todo_app/app.py

```diff
@@ -1,7 +1,9 @@
 from flask import Flask, render_template, request
 
 from todo_app.flask_config import Config
-from todo_app.data.session_items import add_item, get_items
+#from todo_app.data.session_items import add_item, get_items
```

Comment:
You can delete this and session_items.py now

Reply:
done